src/incubator/WheelPicker/index.tsx

```diff
@@ -21,13 +21,9 @@ export enum WheelPickerAlign {
 
 export interface WheelPickerProps {
   /**
-   * Initial value (doesn't work with selectedValue)
+   * Initial value
    */
   initialValue?: ItemProps | number | string;
-  /**
-   * The current selected value
-   */
-  selectedValue?: ItemProps | number | string;
```

Comment:
This is a breaking change. I know we're in the incubator but I'm thinking that we might want to at least explain to the uses what's wrong. Maybe add a deprecated message or error them when we detect they passed this prop... or we relay on the ts errors alone?

Reply:
We don't have any internal users using this feature, regarding the community - we will add it under `Breaking Changes` in the release notes (and they also can decide not to update to this version until they fix it on their side)

Reply:
SectionsWheelPicker is using this component and it is not in the incubator. If someone passed 'selectedValue' to one of the sections (type WheelPickerProps) it will be affected by this change. How do we deal with that?

Reply:
You are right, but there's no internal use for `SectionsWheelPicker` in one app (we wrap it in private with different components), and for the community it's the same answer as before - we'll update it in the release notes, the ts we'll let them know as well, and they can stay with the previous version until they'll migrate to the other implementation.

Reply:
We usually don't introduce breaking changes in a patch versions. Let's discuss this with Ethan. @ethanshar

Reply:
It's ok, that's why we have the Incubator components.
We can merge this and as Lidor mention we'll add a breaking changes section for this (also mention the SectionWheelPicker component). 

Moreover, this breaking change do offer an easy alternative and the community can also lock version till they handle the change.